tests/run-make/doctests-merge/rmake.rs

```diff
@@ -0,0 +1,47 @@
+extern crate run_make_support;
+
+use run_make_support::{diff, rustc, rustdoc, tmp_dir};
+use std::path::Path;
+
+fn test_and_compare(
+    input_file: &str,
+    stdout_file: &str,
+    edition: &str,
+    should_succeed: bool,
+    dep: &Path,
+) {
+    let mut cmd = rustdoc();
+
+    cmd.input(input_file)
+        .arg("--test")
+        .arg("-Zunstable-options")
+        .arg("--edition")
+        .arg(edition)
```

Comment:
Suggestion: add an `edition()` helper to `Rustdoc` just like the one on `Rustc`.